tests/acceptance/posts-test.js

```diff
@@ -73,15 +73,15 @@ module('Acceptance: title', function(hooks) {
     await visit('/feeds/tomster');
     assert.equal(title(), 'Tomster (@tomster)');
 
-    await click('#zoey');
+    click('#zoey');
```

Comment:
This seems odd, why remove the await here?

Reply:
Ah, sorry this is typo from debugging behaviour in DOM elements used for selectors below.

Reply:
Fixed.